File: protcol/c#/caller/hub_call_clientcaller.cs
/*this caller file is codegen by juggle for c#*/
using System;
using System.Collections;
using System.IO;

namespace caller
{
    public class hub_call_client : juggle.Icaller 
    {
        public hub_call_client(juggle.Ichannel _ch) : base(_ch)
        {
            module_name = "hub_call_client";
        }

        public void call_client(String argv0,String argv1,ArrayList argv2)
        {
            ArrayList _argv = new ArrayList();
            _argv.Add(argv0);
            _argv.Add(argv1);
            _argv.Add(argv2);
            call_module_method("call_client", _argv);
        }

    }
}

File: protcol/js/caller/hub_call_clientcaller.js
/*this caller file is codegen by juggle for js*/
function hub_call_client_caller(ch){
    Icaller.call(this, "hub_call_client", ch);

    this.call_client = function( argv0, argv1, argv2){
        var _argv = [argv0,argv1,argv2];
        this.call_module_method.call(this, "call_client", _argv);
    }

}
(function(){
    var Super = function(){};
    Super.prototype = Icaller.prototype;
    hub_call_client_caller.prototype = new Super();
})();
hub_call_client_caller.prototype.constructor = hub_call_client_caller;

